docs/core/testing/unit-testing-with-mstest.md

```diff
@@ -0,0 +1,211 @@
+---
+title: Use MSTest with .NET Core
+description: How to use MSTest with .NET Core
+keywords: MSTest, .NET, .NET Core
+author: ncarandini
+ms.author: wiwagn
+ms.date: 02/10/2017
+ms.topic: article
+ms.prod: .net-core
+ms.devlang: dotnet
+ms.assetid: ed447641-3e85-4e50-b7ed-004630048a3e
+---
+
+# Unit testing with MSTest and .NET Core
+
+## Creating the Projects
+
+[Writing Libraries with Cross Platform Tools](../tutorials/libraries.md)
+has information on organizing multi-project solutions for both the
+source and the tests. This article follows those conventions. The
+final project structure will be something like this:
+
+```
+/unit-testing-using-mstest
+|__/PrimeService
+   |__Source Files
+   |__PrimeService.csproj
+|__/PrimeService.Tests
+   |__Test Files
+   |__PrimeService.csproj
+```
+
+### Creating the source project
+
+Open a shell window. 
+Start in the *unit-testing-using-mstest* directory, create the *PrimeService* directory.
+Make *PrimeService* the current directory, and run `dotnet new classlib` to create the source
+project.
+
+Rename *Class1.cs* as *PrimeService.cs*. To use test-driven development (TDD), you'll create a failing implementation of the
+`PrimeService` class:
+
+```cs
+using System;
+
+namespace Prime.Services
+{
+    public class PrimeService
+    {
+        public bool IsPrime(int candidate) 
+        {
+            throw new NotImplementedException("Please create a test first");
+        } 
+    }
+}
+```
+
+### Creating the test project
+
+Next, cd back into the 'unit-testing-using-mstest' directory, and create the *PrimeServices.Tests* directory.
```

Comment:
Fixed in PR #1531.

@blackdwarf: @mairaw is right, the style guide calls for folder names to be italic, not code style.